src/Chapter2.hs

```diff
@@ -336,8 +336,9 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
-
+subList i j xs
+  | i < j && i > 0 && j > 0 =  take (j - i + 1) (drop i xs)
```

Comment:
Awesome!
Though if i ==j, it could return one-element list (in case if it present):
```suggestion
  | i <= j && i > 0 && j > 0 =  take (j - i + 1) (drop i xs)
```